previews/primer/alpha/text_field_preview.rb

```diff
@@ -68,6 +71,14 @@ def playground(
           }
         end
 
+        # You're required to specify a leading visual if a leading spinner is requested
+        if leading_spinner && !leading_visual_icon
+          system_arguments[:leading_visual] = {
+            icon: :search,
+            size: :small
+          }
+        end
+
```

Comment:
I can see the spinner in the markup, but can't get it to trigger, does it work in preview without any setup?

Reply:
No, you have to trigger it yourself (for now) by doing something like `document.querySelector("primer-text-field").showLeadingSpinner()`.